spec/factories/organisation.rb

```diff
@@ -11,6 +11,13 @@
     holds_own_stock { true }
   end
 
+  factory :organisation_la do
```

Comment:
Why are we adding this?

Reply:
This came out of a merge conflict, I will remove it